src/ml-engine/main_app/main.py

```
@@ -3,9 +3,11 @@
 from starlette.responses import JSONResponse
 from starlette.endpoints import HTTPEndpoint
 from jsonschema import validate, ValidationError
+from main_app.models.BlackLittermanYieldModel import BlackLittermanYieldModel
```

Ruff (0.8.2)

`main_app.models.BlackLittermanYieldModel.BlackLittermanYieldModel` imported but unused

Remove unused import: `main_app.models.BlackLittermanYieldModel.BlackLittermanYieldModel`

(F401)